doc/configuration.rst

```diff
@@ -481,6 +481,7 @@ We recommend that you copy this set of exit codes to your ``luigi.cfg`` file:
   # They are in increasing level of severity (for most applications)
   already_running=10
   missing_data=20
+  complete_failed=20
```

Comment:
Perhaps use a different number? I suggest 41. 41 because it's somewhat similar to "complete_failed" (has number 40).